Tasks/GooglePlayHalt/google-play-halt.ps1

```diff
@@ -0,0 +1,13 @@
+param (
```

Comment:
Could you please remove powershell handler? We are planning to remove it for other google play tasks - since it's out of support at the moment.

Reply:
Powershell handler has been removed.